Directory.Build.targets

```diff
@@ -30,6 +30,12 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(MSBuildProjectName.EndsWith('Tests'))">
```

Comment:
Someone will eventually call a project FooTest and then include these manually. 😆 I don't think we need to do anything about it.